cmd/provider_cmd_datadog.go

```diff
@@ -38,6 +38,7 @@ func newCmdDatadogImporter(options ImportOptions) *cobra.Command {
 	baseProviderFlags(cmd.PersistentFlags(), &options, "monitors,users", "monitor=id1:id2:id4")
 	cmd.PersistentFlags().StringVarP(&apiKey, "api-key", "", "", "YOUR_DATADOG_API_KEY or env param DATADOG_API_KEY")
 	cmd.PersistentFlags().StringVarP(&appKey, "app-key", "", "", "YOUR_DATADOG_APP_KEY or env param DATADOG_APP_KEY")
+	cmd.PersistentFlags().StringVarP(&apiUrl, "api-url", "", "", "YOUR_DATADOG_API_URL or env param DATADOG_HOST")
```

Comment:
It's a bit weird to have the URL named HOST, but we seem to do that everywhere, including terraform, so that looks good.

Reply:
Thanks and I agree its a bit odd but we do follow that format.